package.json

```diff
@@ -180,7 +180,7 @@
   "bundlesize": [
     {
       "path": "./dist/styled-components.min.js",
-      "maxSize": "16kB"
+      "maxSize": "16.5kB"
```

Comment:
Hmm why'd we have to bump this? Aren't we using less code now?

Reply:
The check was failing pretty consistently, but I'm not sure where the extra code came from. I guess the typecheck @kitten suggested (`(mergedTheme === null || Array.isArray(mergedTheme) || typeof mergedTheme !== 'object')`) is slightly longer than the original `isPlainObject(mergedTheme)`, but I'm not sure if that would've been sufficient to break the test.